app/src/test/java/io/crate/node/NodeSettingsTest.java

```diff
@@ -129,4 +131,55 @@ public void testDefaultPaths() {
         ));
         assertTrue(node.settings().get(PATH_LOGS_SETTING.getKey()).endsWith("logs"));
     }
+
+    @Test
+    public void test_legacy_table_function_column_naming() throws Exception {
+        try (Session session = sqlOperations.newSystemSession()) {
+            SQLResponse response = SQLTransportExecutor.execute(
+                "select col1 from unnest([1])", null, session)
+                .get(SQLTransportExecutor.REQUEST_TIMEOUT.millis(), TimeUnit.MILLISECONDS);
+            assertThat(response.rows()[0][0], is(1));
+
+            response = SQLTransportExecutor.execute(
+                "select col1 from generate_series(1,4)", null, session)
+                .get(SQLTransportExecutor.REQUEST_TIMEOUT.millis(), TimeUnit.MILLISECONDS);
+            assertThat(printedTable(response.rows()), is("1\n2\n3\n4\n"));
+
+            response = SQLTransportExecutor.execute(
+                "select col1 from generate_series(1::long,4::long)", null, session)
+                .get(SQLTransportExecutor.REQUEST_TIMEOUT.millis(), TimeUnit.MILLISECONDS);
+            assertThat(printedTable(response.rows()), is("1\n2\n3\n4\n"));
+
+            response = SQLTransportExecutor.execute(
+                "select col1 from generate_series(1,4,1)", null, session)
+                .get(SQLTransportExecutor.REQUEST_TIMEOUT.millis(), TimeUnit.MILLISECONDS);
+            assertThat(printedTable(response.rows()), is("1\n2\n3\n4\n"));
+
+            response = SQLTransportExecutor.execute(
+                "select col1 from generate_series(1::long,4::long,1::long)", null, session)
+                .get(SQLTransportExecutor.REQUEST_TIMEOUT.millis(), TimeUnit.MILLISECONDS);
+            assertThat(printedTable(response.rows()), is("1\n2\n3\n4\n"));
+
+            response = SQLTransportExecutor.execute(
+                "select col1 from generate_series('2019-01-01 00:00'::timestamp, '2019-01-04 00:00'::timestamp, '30 hours'::interval)",
+                null,
+                session)
+                .get(SQLTransportExecutor.REQUEST_TIMEOUT.millis(), TimeUnit.MILLISECONDS);
+            assertThat(printedTable(response.rows()), is("1546300800000\n1546408800000\n1546516800000\n"));
+
+            response = SQLTransportExecutor.execute(
+                "select groups from regexp_matches('alcatraz', 'traz')",
+                null,
+                session)
+                .get(SQLTransportExecutor.REQUEST_TIMEOUT.millis(), TimeUnit.MILLISECONDS);
+            assertThat(printedTable(response.rows()), is("[traz]\n"));
+
+            response = SQLTransportExecutor.execute(
+                "select groups from regexp_matches('99 bottles of beer on the wall', '\\d{2}\\s(\\w+).*', 'ixU')",
+                null,
+                session)
+                .get(SQLTransportExecutor.REQUEST_TIMEOUT.millis(), TimeUnit.MILLISECONDS);
+            assertThat(printedTable(response.rows()), is("[bottles]\n"));
```

Comment:
As we want to test the return `columnName` here, I think the test could be simplified by e.g.
```
            SQLResponse response = SQLTransportExecutor.execute(
                "select * from unnest([1])", null, session)
                .get(SQLTransportExecutor.REQUEST_TIMEOUT.millis(), TimeUnit.MILLISECONDS);
            assertThat(response.cols(), arrayContaining("col1"));
```

Additionally, I think that 1 test case per function or available function signature is enough.